app/assets/javascripts/quotingPanel.js

```diff
@@ -31,16 +38,22 @@ class QuotingPanel {
         secondaryAddressNode.appendChild(div);
       });
       document.getElementById("displayAddress").classList.remove("hidden");
+      document.getElementById("step2").classList.remove("hidden");
     }
   }
 
-  handlePolygonChanged (polygons) {
+  handlePolygonChanged(polygons) {
+    document.getElementById("step2").classList.add("hidden");
```

Comment:
Might want to consider adding `showElement`/`hideElement` methods
```javascript
showElement(id) {
  document.getElementById("step2").classList.remove("hidden");
}

hideElement(id) {
  document.getElementById("step2").classList.add("hidden");
}
```

Reply:
add hideNode in dom.js  and change on all files as well as shodeNode

Reply:
done